src/commands/build.ts

```diff
@@ -12,6 +12,11 @@ import {ArgDescriptor} from 'command-line-args';
 import {Command} from './command';
 import * as logging from 'plylog';
 
+// Only import type definitions here, otherwise this line will be included in
+// the JS output, triggering  the entire build library & its dependencies to
+// be loaded and parsed.
+import {BuildOptions} from '../build/build';
```

Comment:
in addition to the import statement? We still want to use the BuildOptions object in our definitions in this file